orchestrators/InsightOperations/tests/Unit/Coordinators/CoordinatorFakes.php

```
@@ -0,0 +1,300 @@
+                new MetricFactDto("savings", 2500.0),
+            ],
+        );
+    }
+}
+
+final class RiskFactsFake implements
+    RiskInsightFactsQueryInterface,
+    RiskInsightFactsCommandInterface
+{
+    public function factsForRfq(
+        string $tenantId,
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$tenantId'. (undefined)

(UnusedFormalParameter)